src/utils/BaseService.ts

```
@@ -1,70 +1,74 @@
 // src/services/BaseService.ts
-import { AxiosError } from 'axios'
-import { api } from '../libs/axios.ts'
-import { HttpErrorHandler } from './HttpErrorHandler.ts'
-import { ErrorResponse } from '@/commons/interfaces.ts'
+  import { AxiosError } from 'axios'
```

ESLint

[error] Delete `··`

(prettier/prettier)